tests/Microsoft.Identity.Test.Android.UIAutomation/androidTests.cs

```diff
@@ -46,6 +47,7 @@ public void InitializeBeforeTest()
             _app = AppFactory.StartApp(_platform, "com.Microsoft.XFormsDroid.MSAL");
             _xamarinController.Application = _app;
             _mobileTestHelper = new MobileTestHelper(_platform);
+            TestCommon.ResetInternalStaticCaches();
```

Comment:
This is good, but for the "fast run" test, it will not call ResetInternalStatiCaches() before running each individual test.

Reply:
done.